crates/bevy_render/src/view/window.rs

```diff
@@ -161,7 +160,7 @@ pub fn prepare_windows(
             });
 
         let swap_chain_descriptor = wgpu::SurfaceConfiguration {
-            format: TextureFormat::bevy_default(),
+            format: surface.get_supported_formats(&render_adapter)[0],
```

Comment:
This will panic if the returned `Vec` is empty. It would probably be better to still panic, but to panic with a more useful error message.

Reply:
Done! `info!` the `AdapterInfo` then `panic!("No supported texture formats found!")`.